lib/ozfarium_web/live/gallery.ex

```diff
@@ -187,6 +187,37 @@ defmodule OzfariumWeb.Live.Gallery do
     {:noreply, after_visibility_update(socket, ozfa)}
   end
 
+ @impl true
+def handle_event("increment-ep-count", %{"id" => id}, %{assigns: assigns} = socket) do
+  ozfa = Gallery.get_ozfa!(id)
+  user_ozfa = Gallery.find_user_ozfa(ozfa, assigns.current_user)
+
+  current_ep_timestamps = user_ozfa.ep_timestamps || []
+
+  updated_ep_timestamps = [
+    NaiveDateTime.utc_now() | current_ep_timestamps
+  ]
+
+  Gallery.update_user_ozfa(user_ozfa, %{ep_timestamps: updated_ep_timestamps})
+  {:noreply, after_visibility_update(socket, ozfa)}
+end
+
+  
+  @impl true
+  def handle_event("decrement-ep-count", %{"id" => id}, %{assigns: assigns} = socket) do
```

Comment:
I think it will be good to combine these handlers into one `"count-ep"` with an additional value `action` = `increment` or `decrement`.
And abstract increment/decrement update into:
`Gallery.count_ep(ozfa_id, user, action)` to keep `GalleryLive` lean.

Reply:
done